### ISSUE TITLE
Since 0.10.0 not filled attributes or even relationships are rendered in attributes

### ISSUE THREAD

**Issue by jimirocks** [toedter/spring-hateoas-jsonapi#19]
Having the object
```kotlin
@JsonInclude(JsonInclude.Include.NON_NULL)
public class Entity {
  @JsonApiId
  lateinit var id: String

  var title: String? = null

  @JsonApiRelationships("toOneRelation")
  var toOneRelation: ToOneRelation? = null
}
```

And filling the builder like
```kotlin
val entity = Entity()
entity.id = "eId"
JsonApiModelBuilder.jsonApiModel().model(entity).build()
```

Produces
```json
{"data":{"id":"123","type":"entity","attributes":{"toOneRelation":null,"title": null}}}
```

In the 0.9.0 version, there were no `null`s and also no relations in attributes (since my code intentionally doesnot fill the relation properties it's basically about not including `null` into rendering)

Is this a BUG or is it expetced to somehow create model differently?

**Comment by toedter** [comment]
Thanks for reporting this. 
@JsonApiRelationships is only interpreted during deserialization, see https://toedter.github.io/spring-hateoas-jsonapi/0.10.0/reference/#Creating

But it looks like a bug, that `@JsonInclude(JsonInclude.Include.NON_NULL)` is ignored.
I'll look into this.

**Comment by toedter** [comment]
Fixed in the current snapshot, the fix will be in the next release.

**Comment by toedter** [comment]
released in v0.11.0

**Comment by jimirocks** [comment]
nice, thank you
